libs/MVS/SceneDensify.cpp

```diff
@@ -1962,9 +1963,17 @@ void Scene::DenseReconstructionEstimate(void* pData)
 			}
 			// try to load already compute depth-map for this image
 			if (depthmapComputed && data.nFusionMode >= 0) {
+				const std::string depthMapName(ComposeDepthFilePath(data.scene.images[idx].ID, "dmap"));
+				cv::Size depthMapSize;
+				if (GetDepthMapHeaderSize(depthMapName, depthMapSize) && depthMapSize != data.scene.images[idx].GetSize()) {
+					depthData.Load(depthMapName);
+					DepthMap& depthMap = depthData.depthMap;
+					cv::resize(depthMap, depthMap, data.scene.images[idx].GetSize(), 0, 0, cv::INTER_NEAREST);
```

Comment:
resizing only the depth-map is not enough, you should resize all images inside, ex normal-map, etc